torchvision/models/detection/transform.py

```diff
@@ -76,17 +91,12 @@ def torch_choice(self, l):
     def resize(self, image, target):
         # type: (Tensor, Optional[Dict[str, Tensor]])
         h, w = image.shape[-2:]
-        im_shape = torch.tensor(image.shape[-2:])
-        min_size = float(torch.min(im_shape))
-        max_size = float(torch.max(im_shape))
         if self.training:
             size = float(self.torch_choice(self.min_size))
         else:
             # FIXME assume for now that testing uses the largest scale
             size = float(self.min_size[-1])
-        scale_factor = size / min_size
-        if max_size * scale_factor > self.max_size:
-            scale_factor = self.max_size / max_size
+        scale_factor = compute_scale_factor(image, size, float(self.max_size))
```

Comment:
This will probably not work for torchscript because you marked it as  as `torch.jit.unused`.

What I think you'll need to do is instead  to have two functions, one for torchscript and one for the ONNX codepath, which gets activated when the code is tracing.

Reply:
I had to mark it as unused to be able to do "from torch.onnx import operators" in the method.
With torch.jit.script, this was not possible. But I can take the import out and mark is as torch.jit.script. Or create a branch for ONNX

Reply:
Please create a branch for ONNX, otherwise this will add a hard-dependency on ONNX in torchvision.

I'm thinking about something like
```python
if torchvision._is_tracing():
    scale_factor = _compute_scale_factor_onnx(...)
else:
    scale_factor = _compute_scale_factor(...)
```